src/ReactiveUI/Platforms/netstandard2.0/PlatformRegistrations.cs

```diff
@@ -16,7 +17,13 @@ public class PlatformRegistrations : IWantsToRegisterStuff
         /// <inheritdoc/>
         public void Register(Action<Func<object>, Type> registerFunction)
         {
-            throw new Exception("You are referencing the Portable version of ReactiveUI in an App. Please change your reference to the specific version for your platform found here: https://reactiveui.net/docs/getting-started/installation");
```

Comment:
Probably worth leaving a log message here for those newbies who are experiencing issues with threading and start looking into the debug output to find answers about what's going wrong with scheduling. Although not so critical and shouldn't stop the PR :+1: